Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -200,12 +215,12 @@ theorem isBlock_orbit (a : X) : IsBlock G (orbit G a) :=
 variable (X)
 
 /-- The full set is a (trivial) block -/
-theorem isFixedBlock_top : IsFixedBlock G (⊤ : Set X) :=
```

Comment:
Please add the deprecated aliases

Reply:
Why not, but nobody ever imported that file, so I do not really see the point of adding a deprecated alias.

Reply:
Anyway, I did it. (Maybe that will help me…)